lib/offsite_payments/integrations/pay_vector.rb

```diff
@@ -0,0 +1,776 @@
+        @@countries << {:iso_code => 239, :country_short_2 => "GS", :country_short_3 =>"SGS", :country_name => "South Georgia and the South Sandwich Islands", :list_priority => 0}
+        @@countries << {:iso_code => 654, :country_short_2 => "SH", :country_short_3 =>"SHN", :country_name => "Saint Helena", :list_priority => 0}
+        @@countries << {:iso_code => 744, :country_short_2 => "SJ", :country_short_3 =>"SJM", :country_name => "Svalbard and Jan Mayen", :list_priority => 0}
+        @@countries << {:iso_code => 90, :country_short_2 => "SB", :country_short_3 =>"SLB", :country_name => "Solomon Islands", :list_priority => 0}
+        @@countries << {:iso_code => 694, :country_short_2 => "SL", :country_short_3 =>"SLE", :country_name => "Sierra Leone", :list_priority => 0}
+        @@countries << {:iso_code => 222, :country_short_2 => "SV", :country_short_3 =>"SLV", :country_name => "El Salvador", :list_priority => 0}
+        @@countries << {:iso_code => 674, :country_short_2 => "SM", :country_short_3 =>"SMR", :country_name => "San Marino", :list_priority => 0}
+        @@countries << {:iso_code => 706, :country_short_2 => "SO", :country_short_3 =>"SOM", :country_name => "Somalia", :list_priority => 0}
+        @@countries << {:iso_code => 666, :country_short_2 => "PM", :country_short_3 =>"SPM", :country_name => "Saint Pierre and Miquelon", :list_priority => 0}
+        @@countries << {:iso_code => 688, :country_short_2 => "RS", :country_short_3 =>"SRB", :country_name => "Serbia", :list_priority => 0}
+        @@countries << {:iso_code => 678, :country_short_2 => "ST", :country_short_3 =>"STP", :country_name => "Sao Tome and Principe", :list_priority => 0}
+        @@countries << {:iso_code => 740, :country_short_2 => "SR", :country_short_3 =>"SUR", :country_name => "Suriname", :list_priority => 0}
+        @@countries << {:iso_code => 703, :country_short_2 => "SK", :country_short_3 =>"SVK", :country_name => "Slovakia", :list_priority => 0}
+        @@countries << {:iso_code => 705, :country_short_2 => "SI", :country_short_3 =>"SVN", :country_name => "Slovenia", :list_priority => 0}
+        @@countries << {:iso_code => 752, :country_short_2 => "SE", :country_short_3 =>"SWE", :country_name => "Sweden", :list_priority => 0}
+        @@countries << {:iso_code => 748, :country_short_2 => "SZ", :country_short_3 =>"SWZ", :country_name => "Swaziland", :list_priority => 0}
+        @@countries << {:iso_code => 690, :country_short_2 => "SC", :country_short_3 =>"SYC", :country_name => "Seychelles", :list_priority => 0}
+        @@countries << {:iso_code => 760, :country_short_2 => "SY", :country_short_3 =>"SYR", :country_name => "Syrian Arab Republic", :list_priority => 0}
+        @@countries << {:iso_code => 796, :country_short_2 => "TC", :country_short_3 =>"TCA", :country_name => "Turks and Caicos Islands", :list_priority => 0}
+        @@countries << {:iso_code => 148, :country_short_2 =>
```

Comment:
If this is the only method that uses the above countries, could you not simplify `@@countries` to be a hash that uses the `country_short_2` as the key, and the `iso_code` as the value and just do a simple lookup?